packages/graphback-runtime-mongodb/tests/queryBuilderTest.ts

```diff
@@ -18,10 +18,12 @@ describe('MongoDBDataProvider Advanced Filtering', () => {
       @model
       """
       type Post {
-      id: ID!
+      _id: GraphbackObjectID
```

Comment:
This will mean that the [client tempalate](https://github.com/aerogear/graphback/tree/2b355e62f73aae04de10878701e4fb595fcfb482/templates/ts-react-apollo-client) will need some updates i.e running a `yarn graphback generate` once downloaded for it to work. We have that in outputted in the console, but I thought I could point out this change.